add-domain

```diff
@@ -1094,3 +1094,4 @@ beritapb.com
 sgp1.digitaloceanspaces.com
 cpanel01wh.bkk1.cloud.z.com
 dispatchweekly.com
+skimswanp.com
```

Comment:
Add it to the `add-wildcard-domain` then all subdomains it blacklisted as well


Thanks for a good and detailed report